LibGAP.jl/src/gap_to_julia.jl

```diff
@@ -101,8 +101,9 @@ gap_to_julia(type_obj,obj,recursion_dict) = gap_to_julia(type_obj,obj)
 
 ## Convert GAP string to Array{UInt8,1}
 function gap_to_julia( ::Type{Array{UInt8,1}}, obj :: MPtr )
+    ## convert strings to uint8 lists, if wished for
```

Comment:
```suggestion
    ## convert strings to uint8 lists, if requested
```
I find it a bit odd that this comes before the case that handles non-strings. Perhaps reverse the logic, i.e, `if IsStringRep ... else ` ?